plugins/org.jkiss.dbeaver.ext.denodo.ui/src/org/jkiss/dbeaver/ext/denodo/views/DenodoConnectionPage.java

```diff
@@ -48,7 +48,7 @@
 /**
  * DenodoConnectionPage
  */
-public class DenodoConnectionPage extends ConnectionPageAbstract implements IDialogPageProvider, IDataSourceConnectionTester {
+public class    DenodoConnectionPage extends ConnectionPageAbstract implements IDialogPageProvider, IDataSourceConnectionTester {
```

Comment:
I'm not sure that we need this change.

Reply:
Sure, I will change it.

Reply:
Done f0a59174e71abf23909d75fab3f59390e341f7b7